### ISSUE TITLE
node wont install or update 

### ISSUE THREAD

**Issue by TechMasterfury** [Botspot/pi-apps#729]
everytime i install node js it finishes with errors saying this ""____--
```

Installing Node.js with install-32 script
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 13527  100 13527    0     0  64109      0 --:--:-- --:--:-- --:--:-- 64414
=> nvm is already installed in /home/pi/.config/nvm, trying to update using git
=> => Compressing and cleaning up git repository

=> nvm source string already in /home/pi/.bashrc
=> bash_completion source string already in /home/pi/.bashrc
npm WARN npm npm does not support Node.js v10.24.0
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/
=> Close and reopen your terminal to start using nvm or run the following to use it now:

export NVM_DIR="$HOME/.config/nvm"
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
sed: can't read /home/pi/.nvm/nvm.sh: No such file or directory
/home/pi/pi-apps/apps/Node.js/install-32: line 32: nvm: command not found

Failed to install Node.js with install-32 script!
\u25b2 Need help? Copy the above error output or take a screenshot.
Please ask on Github: https://github.com/Botspot/pi-apps/issues/new/choose
Or on Discord: https://discord.gg/RXSTvaUvuu
```

whats the issue

**Comment by Itai-Nelken** [comment]
Its a known probkem that will be fixed when #708 is merged. For now wait until this issue is closed before updating node.js

**Comment by Itai-Nelken** [comment]
@TechMasterfury the issue should be fixed. I'll close this issue. if you have any problems, send them in #789
